### ISSUE TITLE
Minify CI4

### ISSUE THREAD

**Issue by bedrijfsportaal** [slav123/CodeIgniter-minify#62]
Hi,

First of all, I really appreciate the effort you put in this GIT project to help us all make CI faster.
Currently i'm working with CI4 to see if my application is compatible, but in CI3 i'm using your minifier.

Do you have plans on creating this for CI4 also?

Kind regards,

**Comment by michalsn** [comment]
Yes. I already have an alpha version somewhere on the hard drive... I just need to find some time to look at this new code again and make some tests.

**Comment by ayafaheem** [comment]
Hi dear, 
Thanks for your efforts, i make integrate for minfiy library with my project but it's not work correctly for me , no combine file appear to me why ?   
There are any other configuration i need to do ?

**Comment by michalsn** [comment]
@ayafaheem current version doesn't support CI4 - sorry.
